File: frontend/ai_seo/scripts/extract_tabs.py
import re

with open('frontend/index.php', 'r', encoding='utf-8') as f:
    content = f.read()

# Tab 3 extraction
tab3_match = re.search(r'(<section class="tab-panel" id="tab-3">.*?</section><!-- YENİ 4\. SEKME)', content, re.DOTALL)
if tab3_match:
    tab3_content = tab3_match.group(1).replace('<!-- YENİ 4. SEKME', '')
    with open('frontend/ai_seo/views/tab_ai_seo.php', 'w', encoding='utf-8') as f:
        f.write(tab3_content)
    
    # Replace in content
    content = content.replace(tab3_match.group(1), "<?php include 'ai_seo/views/tab_ai_seo.php'; ?>\n<!-- YENİ 4. SEKME")

# Tab 4 extraction
tab4_match = re.search(r'(<section class="tab-panel" id="tab-4">.*?</section>)\s*(?=</div>\s*</main>)', content, re.DOTALL)
if tab4_match:
    tab4_content = tab4_match.group(1)
    with open('frontend/ai_seo/views/tab_todos.php', 'w', encoding='utf-8') as f:
        f.write(tab4_content)
    
    # Replace in content
    content = content.replace(tab4_match.group(1), "<?php include 'ai_seo/views/tab_todos.php'; ?>")

with open('frontend/index.php', 'w', encoding='utf-8') as f:
    f.write(content)

print("Extraction complete")

File: frontend/ai_seo/scripts/extract_tabs.js
const fs = require('fs');

let content = fs.readFileSync('frontend/index.php', 'utf-8');

const tab3Match = content.match(/(<section class="tab-panel" id="tab-3">[\s\S]*?<\/section>)<!-- YENİ 4\. SEKME/);
if (tab3Match) {
    fs.writeFileSync('frontend/ai_seo/views/tab_ai_seo.php', tab3Match[1]);
    content = content.replace(tab3Match[1], "<?php include 'ai_seo/views/tab_ai_seo.php'; ?>\n");
}

const tab4Match = content.match(/(<section class="tab-panel" id="tab-4">[\s\S]*?<\/section>)\s*(?=<\/div>\s*<\/main>)/);
if (tab4Match) {
    fs.writeFileSync('frontend/ai_seo/views/tab_todos.php', tab4Match[1]);
    content = content.replace(tab4Match[1], "<?php include 'ai_seo/views/tab_todos.php'; ?>\n");
}

fs.writeFileSync('frontend/index.php', content);
console.log("Extraction complete");
